pages/web/hosting/htaccess_how_to_protect_wordpress/guide.en-asia.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Tutorial - Using the htaccess file with WordPress"
+slug: use-htaccess-with-wordpress
+excerpt: "Find out how to secure your WordPress blog with one or more htaccess files"
+section: 'Tutorials'
+order: 
```

Comment:
please add an order, it should not be empty

Reply:
done